src/compiler/diagnosticMessages.json

```diff
@@ -2228,6 +2228,10 @@
         "category": "Error",
         "code": 2716
     },
+    "Class '{0}' is declared to implement class '{1}'. Did you mean 'extends'?": {

```

Comment:
i would rather this stayed `class '{0}' incorrectly implements class '{1}'. Did you mean 'extends'?`

Reply:
@DanielRosenwasser thoughts about this one?

Reply:
I can make a quick change to that effect, I was just using the error message from the issue description.

Reply:
Updated.